src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -66,6 +66,10 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             const til::color tabColor{ _TabColor.Value() };
             ss << fmt::format(L"tabColor: {}, ", tabColor.ToHexString(true));
         }
+        if (!_ColorScheme.empty())
+        {
+            ss << fmt::format(L"colorScheme: {}, ", _ColorScheme);

```

Comment:
Might it make sense to use FMT_COMPILE here  (and the one below) instead similar to https://github.com/microsoft/terminal/pull/9581 or would it make more sense to do all the params at once in a separate PR?

Reply:
Probably makes sense to do them all in a single pass IMO